packages/node/src/stack-parser.ts

```diff
@@ -0,0 +1,117 @@
+import { basename, dirname, StackLineParser } from '@sentry/utils';
+
+/** Gets the module */
+function getModule(filename: string | undefined): string | undefined {
+  if (!filename) {
+    return;
+  }
+
+  const base = `${
+    (require && require.main && require.main.filename && dirname(require.main.filename)) || global.process.cwd()
```

Comment:
We can use optional chaining in node land since we down-compile when we transpile from TS -> JS

Reply:
I had to revert the optional chaining because it broke a webpack test somewhere. 

I'm going to have to some back the require issue later!

Reply:
Interesting, wonder what's going on. Is it the tests we have in https://github.com/getsentry/sentry-javascript/tree/master/packages/node/test/manual? 

I think this is partly why we have utils like https://github.com/getsentry/sentry-javascript/blob/6809dd2ae41f81e3b5d3e3c8076f2c6bcb9cfbfb/packages/utils/src/node.ts#L28-L31

Reply:
It was actually the nextjs webpack tests that started failing.

The issue with `dynamicRequire` is that it's only designed to work with `module.require`.

However, `module.require` only gives you access to the require function. It does not include any other properties like `require.main`. This are only available on `require` which isn't even a global.